main.js

```diff
@@ -122,6 +123,8 @@ module.exports = function (options) {
             withBackendAuthentication: true,
             healthChecks: []
         },
+        options = _.merge(defaults, argumentOptions),
+        host = process.env.HOST || options.host,
```

Comment:
Currently host is not being accounted for in the passed in options. This PR addresses that.